core/primitives-core/src/version.rs

```diff
@@ -211,7 +232,13 @@ pub enum ProtocolFeature {
         note = "Was used for protocol versions without relaxed chunk validation which is not supported anymore."
     )]
     _DeprecatedRelaxedChunkValidation,
-    /// This enables us to remove the expensive check_balance call from the runtime.
+    /// This enables us to remove the expensive check_balance call from the
+    /// runtime.
+    ///
+    /// Support for code that does check balances has now been removed.
+    #[deprecated(
+        note = "Was used for protocol versions where we checked balances which is not supported anymore."
+    )]
```

Comment:
btw I'm sad to see check balance go entirely, it was a useful tool, is there any way to restore it to former glory?

Reply:
Hmm... we dropped the code in a previous PR.  If it is useful, we could resurrect the code for test purposes.